.github/workflows/docker.yaml

```diff
@@ -0,0 +1,71 @@
+name: Docker Build HTML Image
```

Comment:
Could you please create me a feature branch in useblocks repo `feature/531_add_docker_support`?
I will do a temporary merge to the feature branch so that I can test this workflow file. I do not think I can test a workflow otherwise on the useblocks CI.